src/models/City.ts

```diff
@@ -0,0 +1,18 @@
+export class  City{
+    id: String;
+    name: string
+    temperature: string;
+    humidity: string;
+
+    constructor(
+        id: string,
+        name: string,
+        temperature: string,
+        humidity: string
+    ){
+        this.id = id,
+        this.name = name,
+        this.temperature = temperature,
+        this.humidity = humidity
```

Comment:
The data types here don't seem right, I don't if the temperature should a string along with the humidity.

Reply:
It was just an example but i think they could be string too